java-ast/tests/pom.xml

```diff
@@ -41,6 +41,12 @@
             <artifactId>windup-exec</artifactId>
             <classifier>forge-addon</classifier>
             <scope>test</scope>
+        </dependency>    
+        <dependency>
+            <groupId>org.eclipse.tycho</groupId>
+            <artifactId>org.eclipse.jdt.core</artifactId>
+            <version>3.10.0.v20140604-1726</version>
+            <scope>test</scope>
```

Comment:
@juangon our CI tests does that on clean environment and didn't fail on that I am sure. But I can give a try on my isolated env too, just for sure.

Wrt locale, if I use my native czech locale i could see the test failures too in past, there is testing of a number in thousands so there are dots/commas in formatted efforts in reports depending what locale can freemarker use.